Dockerfile.Windows

```diff
@@ -1,9 +1,24 @@
-FROM mcr.microsoft.com/windows/servercore:1809 as core
+# Other common ones: 1803-amd64, 1903-amd64
+# full list: curl -L https://mcr.microsoft.com/v2/windows/nanoserver/tags/list
+
+ARG golangTag=1.14.0-windowsservercore-1809
```

Comment:
Is this picking Go 1.14? I'm a bit worried that this is not what we use for Linux (yet) and that keeping this in sync will become a maintenance issue.

But without additional scripting I don't see a way around that.

Reply:
Yeah I wasnt sure what was installed on the build systems you were using. I assumed it would be the latest (we are using the latest 1.13 release for our internal builds here) so I just went ahead and picked 1.14. If you can tell me what release you are using Ill match it up.

Reply:
The canonical definition for each component is in it's .travis.yml file, which typically is a symlink to `release-tools/travis.yml`: https://github.com/kubernetes-csi/node-driver-registrar/blob/b4bdb81f33abf9eff80c194abe67c241a5900f96/release-tools/travis.yml#L9

All other places where we need to pick a Go version are also based on that definition, for example: https://github.com/kubernetes-csi/node-driver-registrar/blob/b4bdb81f33abf9eff80c194abe67c241a5900f96/release-tools/prow.sh#L94-L100

We also warn developers when they use something else: https://github.com/kubernetes-csi/node-driver-registrar/blob/b4bdb81f33abf9eff80c194abe67c241a5900f96/release-tools/verify-go-version.sh#L33-L48

Reply:
> Yeah I wasnt sure what was installed on the build systems you were using.

It's the other way around :grin: In those places that we control (CI), we install what want to use, instead of using what is installed.

Developers are free to diverge from that, but at their own risk.

Reply:
k ive got a commit coming that updates this to 1.13.3